cloudinit/config/cc_reset_rmc.py

```diff
@@ -0,0 +1,167 @@
+# (c) Copyright IBM Corp. 2020 All Rights Reserved
+#
+# Author: Aman Kumar Sinha <amansi26@in.ibm.com>
+#
+# This file is part of cloud-init. See LICENSE file for license information.
+
+
+"""
+Reset RMC
+------------
+**Summary:** reset rsct node id
+
+Reset RMC module is IBM PowerVM Hypervisor specific
+
+Reliable Scalable Cluster Technology (RSCT) is a set of software components,
+that  together provide a comprehensive clustering environment (RAS features)
+for IBM PowerVM based virtual machines. RSCT includes the Resource monitoring
+and control (RMC) subsystem. RMC is a generalized framework used for managing,
+monitoring, and manipulating resources. RMC runs as a daemon process on
+individual machines and needs creation of unique node id and restarts
+during VM boot.
+More details refer
+https://www.ibm.com/support/knowledgecenter/en/SGVKBA_3.2/admin/bl503_ovrv.htm
+
+This module handles
+- creation of the unique RSCT node id to every instance/virtual machine
+  and ensure once set, it isn't changed subsequently by cloud-init.
+  In order to do so, it restarts RSCT service.
+
+Prerequisite of using this module is to install RSCT packages.
+
+**Internal name:** per instance
+
+**Supported distros:** rhel, sles and ubuntu
+
+"""
+
+import json
+import os
+import time
+
+from cloudinit.settings import PER_INSTANCE
+from cloudinit import util
+from cloudinit import subp
+
+frequency = PER_INSTANCE
+PIDOF = 'pidof'
+RMCCTRL = '/usr/sbin/rsct/bin/rmcctrl'
```

Comment:
@Aman306 , we should use the new path namely /opt/rsct/bin . /usr/sbin/rsct/bin/ is the old path and now a symbolic link to  /opt/rsct/bin. As pointed out in a previous comment, adding /opt/rsct/bin to PATH variable also should work fine. I'd recommend adding a comment above, mentioning the full path as /opt/rsct/bin and that the user has to set this under the PATH variable and use rmcctrl instead of the full path.

https://www.ibm.com/support/knowledgecenter/SGVKBA_3.2/admin/admin_pdf.pdf

_The rmcctrl command is located in /opt/rsct/bin. Add this directory to your PATH, or specify the
full path on the command line_

@smoser @otubo FYI.